<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>641</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Port COM</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>175</width>
        <height>174</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="combobox1"/>
       </item>
       <item>
        <widget class="QPushButton" name="openport1button">
         <property name="text">
          <string>OPEN PORT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeport1button">
         <property name="text">
          <string>CLOSE PORT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Move</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>Position X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinX">
           <property name="minimum">
            <number>-200</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelY">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Position Y:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinY">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelZ">
           <property name="text">
            <string>Position Z:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinZ">
           <property name="minimum">
            <number>400</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>MOVE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>211</width>
        <height>136</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="labelHOME">
           <property name="text">
            <string>Home Position: [0,0,400]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendHPOS">
           <property name="text">
            <string>HOME POSITION</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelVEL">
               <property name="text">
                <string>MAX Velocity [rpm]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinVEL">
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="sendVEL">
             <property name="text">
              <string>SET MAX VELOCITY</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>160</width>
        <height>103</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="sendEN">
         <property name="text">
          <string>ENABLE SERVOS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPFB">
         <property name="text">
          <string>POSITION FEEDBACK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendA">
         <property name="text">
          <string>HOMING</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Trajectory</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>161</width>
        <height>103</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="sendLINEAR">
         <property name="text">
          <string>LINEAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPOLY3">
         <property name="text">
          <string>CUBIC POLYNOMIAL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPOLY5">
         <property name="text">
          <string>QUINTIC POLYNOMIAL</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
